tests/catalog/test_sql.py

```diff
@@ -1173,6 +1173,8 @@ def test_update_namespace_properties(catalog: SqlCatalog, namespace: str) -> Non
         else:
             assert k in update_report.removed
     assert "updated test description" == catalog.load_namespace_properties(namespace)["comment"]
```

Comment:
nit: I think it'll be good to merge these 3 lines and check for all the namespace properties returned by `catalog.load_namespace_properties(namespace)`

Reply:
merged these checks with the updated_report check

Reply:
nit: I feel the test was more readable before. 

I think having two separate checks is preferred. The first checks the return value from `catalog.update_namespace_properties` and then checks the actual values from `catalog.load_namespace_properties`. 

I also like checks with hardcoded values 
```
catalog.load_namespace_properties(namespace) == {
    "comment": "updated test description",
   ...
"
so that I know exactly what `catalog.load_namespace_properties` returns as I'm reading the test.

Reply:
make sense. updated code. pls review it again. @kevinjqliu